woocommerce/woocommerce-android
# Do not show delta tag when one revenue delta calculation is not valid

### Description
This pull request hides the delta tag when one of the revenues is 0 due to an Infinite calculation.

### Testing instructions

1. Open app compiled in debug mode
2. Click on the analytics section in the bottom navigation bar
3. Select a date range with the calendar that has 0 revenue data
4. Delta tag should not be shown

### Images/gif

<img src="https://user-images.githubusercontent.com/572601/145540560-827bc6c5-882e-493e-80c7-aa39075fcf97.png" width="380" height="1000">

<img src="https://user-images.githubusercontent.com/572601/145540569-5fe79f47-40fa-4692-8ce2-ce7d3a9d9336.png" width="380" height="1000">
